app/javascript/components/editor/FileEditor.tsx

```diff
@@ -123,6 +119,10 @@ export function FileEditor({
     editor.setModel(filesRef.current[0].model)
 
     editorDidMount({ getFiles, setFiles, openPalette })
+
+    // This fixes a Windows-only issue with the custom code font resulting in the cursor rendering being off
+    // See https://github.com/exercism/website/issues/742#issuecomment-816806513
+    monacoEditor.editor.remeasureFonts()
```

Comment:
@ErikSchierboom How about wrapping it in a `setTimeout` with `0` as the thing to allow for a first-paint to occur before calling it?

Reply:
Worth a shot!

Reply:
Done